addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -157,6 +157,14 @@ <H2>ZAP API New Endpoints:</H2>
 
 <H2>ZAP API Deprecated Endpoints:</H2>
 
+The following endpoints have been superseded by the <a href="https://www.zaproxy.org/docs/desktop/addons/exim/">Import/Export</a> add-on:
```

Comment:
This hasn't been generated yet but I believe it is where things will endup.

Reply:
Should be https://www.zaproxy.org/docs/desktop/addons/import-export/

Reply:
Tks! I’ll fix that up tonight.